src/main/java/com/team4/testingsystem/entities/Question.java

```diff
@@ -0,0 +1,83 @@
+package com.team4.testingsystem.entities;
+
+import javax.persistence.*;
+
+@Entity
+public class Question {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name = "question_body")
+    private String questionBody;
+    private int points;
```

Comment:
Do we need to store the points in a DB?

Reply:
point is the weight of the question from 1 to 10

Reply:
Didn't we decide that we are going to skip adaptive questions for now?

Reply:
now i'm not sure about it

Reply:
![Screenshot_20210709_235617_com.skype.m2.png](https://user-images.githubusercontent.com/86999288/125135230-5283e000-e111-11eb-8301-13fa5fde0968.png)